packages/labs/virtualizer/src/test/support/resize-observer-errors.test.ts

```diff
@@ -26,7 +26,9 @@ async function windowError(message: string) {
   setTimeout(() => {
     throw new CountingError(message);
   });
-  await until(() => errors >= expectedErrorCount);
+  await until(() =>
+    expect(errors).to.be.greaterThanOrEqual(expectedErrorCount)
+  );
```

Comment:
This file needs to import `pass` for the following suggestions.
```suggestion
  await pass(() =>
    expect(errors).to.be.greaterThanOrEqual(expectedErrorCount)
  );
```

Reply:
Fixed